### DISCUSSION TITLE
How to find the date time and browser's timezone ?

### DISCUSSION THREAD

**Discussion by mxav1111** [flet-dev/flet#4175] [Q&A] [answered]
### Question

Hi,
I am unable to find any page variable, that shows browser's time zone and/or browser's current date time.
Just like the one we already have client_ip address/user agent variables.

How do we convert server date time to user's time zone that is accessing flet application via web url ?

### Code sample

```python
N/A
```

### Error message

```bash
N/A
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
It's a valid question you know. I think we could pass a time zone offset as part of the first connection to a server.

More details on that:
* https://stackoverflow.com/questions/13/determine-a-users-timezone
* https://api.dart.dev/stable/3.5.4/dart-core/DateTime/timeZoneOffset.html

**Reply by FeodorFitsner** [reply, marked as answer]
New issue created: https://github.com/flet-dev/flet/issues/4186

**Reply by mxav1111** [reply]
> New issue created: #4186

Thank you.
